api/src/main/java/io/kaoto/backend/api/resource/v2/IntegrationsResource.java

```diff
@@ -0,0 +1,167 @@
+package io.kaoto.backend.api.resource.v2;
+
+import io.kaoto.backend.api.resource.v1.model.Integration;
+import io.kaoto.backend.api.service.deployment.DeploymentService;
+import io.kaoto.backend.api.service.deployment.generator.DeploymentGeneratorService;
+import io.kaoto.backend.api.service.language.LanguageService;
+import io.kaoto.backend.api.service.step.parser.StepParserService;
+import io.kaoto.backend.model.step.Step;
+import org.eclipse.microprofile.openapi.annotations.Operation;
+import org.eclipse.microprofile.openapi.annotations.parameters.Parameter;
+import org.eclipse.microprofile.openapi.annotations.parameters.RequestBody;
+import org.jboss.logging.Logger;
+import org.jboss.resteasy.reactive.server.ServerExceptionMapper;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * 🐱class IntegrationsResource
+ * 🐱relationship compositionOf DeploymentService, 0..1
+ *
+ * This endpoint will return the yaml needed to deploy
+ * the related integration and the
+ * endpoints to interact with deployments.
+ */
+@Path("/v2/integrations")
```

Comment:
@Delawen I created a new resource to have a different path, to keep backward compatibility